src/BaseAvatarUtils.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IAggregatorV3} from "./interfaces/chainlink/IAggregatorV3.sol";
+
+contract BaseAvatarUtils {
+    ////////////////////////////////////////////////////////////////////////////
+    // ERRORS
+    ////////////////////////////////////////////////////////////////////////////
+
+    error StalePriceFeed(uint256 currentTime, uint256 updateTime, uint256 maxPeriod);
+
+    function fetchPriceFromClFeed(IAggregatorV3 _feed, uint256 _maxStalePeriod)
+        internal
+        view
+        returns (uint256 answerUint256_)
+    {
+        (, int256 answer,, uint256 updateTime,) = _feed.latestRoundData();
+
+        if (block.timestamp - updateTime > _maxStalePeriod) {
+            revert StalePriceFeed(block.timestamp, updateTime, _maxStalePeriod);
+        }
+
+        answerUint256_ = uint256(answer);
```

Comment:
Add positive check for future compatibility
It will also avoid more reports in the future

require(answer > 0)

Reply:
good one, added for posterity in 8ebba3e18662daa63063fbcc4b136a3bf313beaa